package.json

```diff
@@ -15,8 +15,9 @@
     "prebuild": "npm run clean:build",
     "build:bundle": "cross-env NODE_OPTIONS=--max-old-space-size=4096 bundle-one-app",
     "build:server": "babel src --out-dir lib --ignore \"**/__mocks__\"",
+    "build:workers": "node scripts/build-workers.js",
     "build:artifacts:statics": "node scripts/build-static-assets-artifact.js",
-    "build": "concurrently \"npm run build:bundle\" \"npm run build:server\" -n build:bundle,build:server --kill-others-on-fail",
+    "build": "concurrently \"npm run build:bundle\" \"npm run build:server\" -n build:bundle,build:server --kill-others-on-fail && npm run build:workers",
```

Comment:
Does this have to be after the `concurrently` call?

Reply:
The workers are built in the `lib/` directory and to avoid overwriting, I've moved out it out to make sure it writes after the lib is built.

Reply:
Yes, it expects to use the build output as part of the build output.